src/vs/workbench/contrib/welcome/gettingStarted/common/gettingStartedContent.ts

```diff
@@ -395,5 +396,158 @@ export const content: GettingStartedContent = [
+					description: localize('gettingStarted.scmSetup.description', "Set up your project to use Git and GitHub integration right inside your editor."),
+					when: 'config.git.enabled && !git.missing && workspaceFolderCount == 0',
+					button: {
+						title: localize('gettingStarted.scmClone.button', "Clone Repository"),
+						command: 'git.clone'
+					},
+					doneOn: { commandExecuted: 'git.clone' },
+					media: {
+						type: 'image', altText: 'Source Control view.', path: {
+							dark: 'dark/scm.png',
+							light: 'light/scm.png',
+							hc: 'light/scm.png'
+						},
+					},
+				},
+				{
+					id: 'scmSetup',
+					title: localize('gettingStarted.scm.title', "Track your code with Git"),
+					description: localize('gettingStarted.scmSetup.description', "Set up your project to use Git and GitHub integration right inside your editor."),
+					when: 'config.git.enabled && !git.missing && workspaceFolderCount != 0 && gitOpenRepositoryCount == 0',
+					button: {
+						title: localize('gettingStarted.scmSetup.button', "Initialize Git Repository"),
+						command: 'git.init'
+					},
+					doneOn: { commandExecuted: 'git.init' },
+					media: {
+						type: 'image', altText: 'Source Control view.', path: {
+							dark: 'dark/scm.png',
+							light: 'light/scm.png',
+							hc: 'light/scm.png'
+						},
+					},
+				},
+				{
+					id: 'scm',
+					title: localize('gettingStarted.scm.title', "Track your code with Git"),
+					description: localize('gettingStarted.scm.description', "Manage common Git and GitHub tasks right inside your editor."),
+					when: 'config.git.enabled && !git.missing && workspaceFolderCount != 0 && gitOpenRepositoryCount != 0 && activeViewlet != \'workbench.view.scm\'',
+					button: {
+						title: localize('gettingStarted.scm.button', "Open Source Control"),
+						command: 'workbench.view.scm'
+					},
+					doneOn: { commandExecuted: 'workbench.view.scm.focus' },
+					media: {
+						type: 'image', altText: 'Source Control view.', path: {
+							dark: 'dark/scm.png',
+							light: 'light/scm.png',
+							hc: 'light/scm.png'
+						},
+					},
+				},
+				{
+					id: 'tasks',
+					title: localize('gettingStarted.tasks.title', "Automate your project tasks"),
+					when: 'workspaceFolderCount != 0',
+					description: localize('gettingStarted.tasks.description', "Configure your workflows as tasks and enjoy the fully integrated experience of running scripts and monitoring results."),
+					button: {
+						title: localize('gettingStarted.tasks.button', "Run Auto-detected Tasks"),
+						command: 'workbench.action.tasks.runTask'
+					},
+					doneOn: { commandExecuted: 'workbench.action.tasks.runTask' },
+					media: {
+						type: 'image', altText: 'Task runner.', path: {
+							dark: 'dark/tasks.png',
+							light: 'light/tasks.png',
+							hc: 'light/tasks.png'
+						},
+					},
+				},
+				{
+					id: 'shortcuts',
+					title: localize('gettingStarted.shortcuts.title', "Assign some hot keys"),
```

Comment:
"Assign some hotkeys" -> "Customize your shortcuts"
For description: "When you've found your favorite commands, assign them a keyboard shortcut for instant access"

Reply:
I had those before as well. _Hotkeys_ was my cool spin 😎. Will go with yours for now as more straightforward.